client/my-sites/domains/components/domain-and-plan-package/navigation/index.jsx

```diff
@@ -6,11 +6,12 @@ import './style.scss';
 export default function DomainAndPlanPackageNavigation( props ) {
 	const translate = useTranslate();
 
+	// `goBackLink` will be either wp-admin or My Home, depending on the user's
+	// current admin interface preference.
+	// If unavailable, we fall back to the previous page in history.
 	const goBack = () => {
 		if ( props.goBackLink ) {
```

Comment:
Also, it seems, now, we can remove this condition too
```suggestion
```

Reply:
If we expect `goBackLink` to always be defined in the props, then we should enforce it. I suggest we do that by converting this file to TS 👍